pkg/api/serialization_test.go

```diff
@@ -131,6 +132,22 @@ func roundTripSame(t *testing.T, group testapi.TestGroup, item runtime.Object, e
 	}
 }
 
+func TestSetControllerConversion(t *testing.T) {
+	rs := &extensions.ReplicaSet{}
+	gv := testapi.Extensions
+	fuzzInternalObject(t, gv.InternalGroupVersion(), rs, rand.Int63())
+
+	data, err := runtime.Encode(gv.Codec(), rs)
+	if err != nil {
+		t.Errorf("unexpected encoding error: %v", err)
+	}
+
+	rc := &v1.ReplicationController{}
+	if err := runtime.DecodeInto(testapi.Default.Codec(), data, rc); err != nil {
```

Comment:
FWIW #26044 is not fixing this issue. See:
https://github.com/kargakis/kubernetes/commits/conversions
http://pastebin.com/hNSyqDiQ (same panic)